docs/examples/taylor.ipynb

```diff
@@ -0,0 +1,120 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# `set_titles_and_labels`\n",
+    "\n",
+    "This notebook is a simple example of the GeoCAT-viz function <a href=\"../user_api/generated/geocat.viz.util.set_titles_and_labels.html#geocat-viz.util.set_titles_and_labels\">set_titles_and_labels</a>."
```

Comment:
Looks like this got missed. 

It looks like it's causing the index on the left to be incorrect as well.  I couldn't find the example at first.
https://geocat-viz--186.org.readthedocs.build/en/186/examples/taylor.html

Reply:
hmm it seems like a commit didn't publish? Because I definitely changed this locally. The link seems off too